knimin/templates/pm_properties.html

```diff
@@ -0,0 +1,172 @@
+<!-- The Plate Mapper module -->
```

Comment:
Can you remove this file from the PR? I don't think it is needed for what you want to add here. Remember that you can work with multiple branches at once. If you don't know how to do that, please talk with @ElDeveloper, @antgonza, me or any other developer since we usually do this all the time.